### PR TITLE
fix(editor): Disable data pinning on multiple output node types

### PR DESCRIPTION
**Pull request by alexgrozav**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

**Review by ivov — DISMISSED** [review]
In future we should think about turning all these lists at the client into settings at node definition level.

File: packages/editor-ui/src/constants.ts [hunk 1]
```diff
@@ -141,7 +142,11 @@ export const NON_ACTIVATABLE_TRIGGER_NODE_TYPES = [
 	EXECUTE_WORKFLOW_TRIGGER_NODE_TYPE,
 ];
 
-export const MULTIPLE_OUTPUT_NODE_TYPES = [IF_NODE_TYPE, SWITCH_NODE_TYPE];
+export const MULTIPLE_OUTPUT_NODE_TYPES = [
```
Comment (mutdmour) [inline packages/editor-ui/src/constants.ts:145]: can we instead check if that node has multiple outputs? the outputs are specified as an array in the node properties.. can we check for that instead?

Comment (alexgrozav) [reply packages/editor-ui/src/constants.ts]: Great idea! Updated.

File: packages/editor-ui/src/constants.ts [hunk 2]
```diff
@@ -141,12 +142,7 @@ export const NON_ACTIVATABLE_TRIGGER_NODE_TYPES = [
 	EXECUTE_WORKFLOW_TRIGGER_NODE_TYPE,
 ];
 
-export const MULTIPLE_OUTPUT_NODE_TYPES = [IF_NODE_TYPE, SWITCH_NODE_TYPE];
-
-export const PIN_DATA_NODE_TYPES_DENYLIST = [
-	...MULTIPLE_OUTPUT_NODE_TYPES,
-	SPLIT_IN_BATCHES_NODE_TYPE,
-];
+export const PIN_DATA_NODE_TYPES_DENYLIST = [SPLIT_IN_BATCHES_NODE_TYPE];
```
Comment (mutdmour) [inline packages/editor-ui/src/constants.ts:145]: agree with @ivov that this can be a node property

File: packages/editor-ui/src/constants.ts [hunk 3]
```diff
@@ -82,6 +82,7 @@ export const BAMBOO_HR_NODE_TYPE = 'n8n-nodes-base.bambooHr';
 export const CALENDLY_TRIGGER_NODE_TYPE = 'n8n-nodes-base.calendlyTrigger';
 export const CODE_NODE_TYPE = 'n8n-nodes-base.code';
 export const CRON_NODE_TYPE = 'n8n-nodes-base.cron';
+export const COMPARE_DATASETS_NODE_TYPE = 'n8n-nodes-base.compareDatasets';
```
Comment (mutdmour) [inline packages/editor-ui/src/constants.ts:85]: this can be deleted since it's not used

**Review by mutdmour — DISMISSED** [review]
LGTM. Thanks for addressing feedback 😊

File: packages/editor-ui/src/constants.ts (hunk 3 above)
Comment (alexgrozav) [reply packages/editor-ui/src/constants.ts]: Done!

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.211.0`

**Pull request merged**